lib/features/family/app/pages.dart

```diff
@@ -1,14 +1,15 @@
 enum FamilyPages {
   profileSelection(path: '/profile-selection', name: 'PROFILE_SELECTION'),
   kidsAvatarSelection(
-      path: '/kids-avatar-selection', name: 'KIDS-AVATAR-SELECTION'),
+      path: 'kids-avatar-selection', name: 'KIDS-AVATAR-SELECTION'),
   parentAvatarSelection(
-      path: '/parent-avatar-selection', name: 'PARENT-AVATAR-SELECTION'),
-  wallet(path: '/wallet', name: 'WALLET'),
-  camera(path: '/camera', name: 'CAMERA'),
-  success(path: '/success', name: 'SUCCESS'),
+      path: 'parent-avatar-selection', name: 'PARENT-AVATAR-SELECTION'),
+  wallet(path: 'wallet', name: 'WALLET'),
+  camera(path: 'camera', name: 'CAMERA'),
+  success(path: 'success', name: 'SUCCESS'),
+  unregisterUS(path: 'unregister-us', name: 'UNREGISTER-US'),
```

Comment:
Im assuming removing the slashes because they are subroutes?

Reply:
Yes made them all subroutes.